docs/index.md

```diff
@@ -1,4 +1,5 @@
 ---
+<<<<<<< HEAD
```

Comment:
Looks like a conflict needs to be resolved first.

Reply:
corrected